src/components/taskDetails/ProgressUpdateCard/ProgressUpdateCard.tsx

```diff
@@ -35,27 +69,38 @@ export default function ProgressUpdateCard({
         setIsTooltipVisible(false);
     }
 
-    function onMoreButtonClick(e: MouseEvent<HTMLElement>) {
-        if (isTitleWrapped) {
-            setTitleToShow(data.completed);
-            setIsTitleWrapped((prevState) => !prevState);
-            return;
-        }
+    function onMoreOrLessButtonClick(
+        e: MouseEvent<HTMLElement>,
+        clickedOnData: ProgressUpdateDataToShow
+    ) {
+        e.stopPropagation();
+        setDataToShowState((prevState) => {
+            const newPrevState = prevState.map((datum) => {
```

Comment:
what is `datum`?

Reply:
it's a param for prevState mapping

Reply:
https://github.com/Real-Dev-Squad/website-status/pull/1111#discussion_r1485924925